src/main/webapp/i18n/de/exam.json

```diff
@@ -452,6 +453,83 @@
                 }
             }
         },
+        "examMonitoring": {
+            "title": "Klausurüberwachung",
+            "description": "Die Klausurüberwachungsansicht erlaubt es die Aktionen der Studierenden während der Klausur zu überwachen und die daraus resultierenden Daten zu analysieren.",
+            "table": {
+                "panelHeader": "Klausurinformationen",
+                "panelDescriptionHeader": "Details",
+                "title": "Titel",
+                "start": "Start",
+                "end": "Ende",
+                "students": "Registrierte Studierende",
+                "exercises": "Aufgaben",
+                "exerciseGroups": "Aufgabengruppen"
+            },
+            "menu": {
+                "overview": {
+                    "title": "Übersicht"
+                },
+                "exercises": {
+                    "title": "Aufgaben",
+                    "description": "Zeigt die aktuelle Teilnahme der Studierenden in den jeweilgen Aufgaben und Aufgabengruppen",
+                    "groups": "Aufgabengruppen"
+                },
+                "activity-log": {
+                    "title": "Aktivitäten",
+                    "description": "Zeigt die vergangenen und aktuellen Aktivitäten der aktiven Studierenden"
+                }
+            },
+            "charts": {
+                "exercise-chart": {
+                    "name": "Aktuelle Anzahl der Studierenden pro Aufgabe gruppiert nach Aufgabengruppen",
+                    "tooltip": "Zeigt die aktuelle Anzahl an Studierenden pro Aufgabe gruppiert nach den Aufgabengruppen.",
```

Comment:
I did not test this yet, but if the tooltip is associated with a help icon, do you think this provides necessary support to users who do not completely understand the title? 😅

Reply:
This actually refers to multiple tooltips in this file

Reply:
mhm i guess tooltips are always useful :D we can extend them in the future :thinking:

Reply:
I agree that this can be delayed to a different PR. And I cannot claim that every tooltip is perfectly explanatory. But keep in mind that you provide this tooltip through a question mark icon. This indicates some additional context or help for a specific term. I think it is a good idea to provide this, especially if you display not-trivial data. Maybe try to illustrate the borders of the data displayed (e.g. how it is created, what is covered) in the tooltip. This helps the user to gain more confidence while using the chart because it is clearer what is actually displayed here and how to work with it.

Reply:
Good idea! Thank you for your review and you suggestions :D those are always welcome :D